src/Ocelot/Configuration/Creator/IMetadataCreator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Ocelot.Configuration.File;
+
+namespace Ocelot.Configuration.Creator;
```

Comment:
Wrong namespace! Should be `Ocelot.Metadata` ❗

Reply:
No, it's a configuration creator. It was like that until now. We should rethink it, but that's not part of the PR.